pandas/core/series.py

```diff
@@ -1885,10 +1885,109 @@ def update(self, other):
     # Reindexing, sorting
 
-    @Appender(generic._shared_docs['sort_values'] % _shared_doc_kwargs)
     def sort_values(self, axis=0, ascending=True, inplace=False,
                     kind='quicksort', na_position='last'):
+        """
+        Sort by the Series values.
+
+        Sort (or order) a Series into ascending or descending order by some criterion.
+
+        Parameters
+        ----------
+        axis : {0 or ‘index’}, default 0
+            Axis to direct sorting.
+        ascending : bool, default True
+            If  `True` sort values into ascending order, otherwise descending.
+        inplace : bool, default False
+            If True, perform operation in-place.
+        kind : {‘quicksort’, ‘mergesort’ or ‘heapsort’}, default ‘quicksort’
+            Choice of sorting algorithm. See also ndarray.np.sort [1]_ for more information. `mergesort` is the only stable
+            algorithm.
+        na_position : {'first' or 'last'}, default 'last'
+             Argument `first` puts NaNs at the beginning, `last` puts NaNs at the end.
 
+        Returns
+        -------
+        Series
+            Series ordered by values.
+
+        See Also
+        --------
+        Series.sort_index : Sort by the Series indices.
+        DataFrame.sort_index : Sort DataFrame by indices.
+        DataFrame.sort_values : Sort by the values along either axis.
+
+        References
+        ----------
+        .. [1] https://docs.scipy.org/doc/numpy/reference/generated/numpy.sort.html
+
+        Examples
+        --------
+        >>> s = pd.Series([np.nan, 1, 232, 323, 1, 2, 3, 45])
+        >>> s
+        0      NaN
+        1      1.0
+        2    232.0
+        3    323.0
+        4      1.0
+        5      2.0
+        6      3.0
+        7     45.0
+        dtype: float64
+
+        Sort values ascending order
+
+        >>> s.sort_values(ascending=True)
+        1      1.0
+        4      1.0
+        5      2.0
+        6      3.0
+        7     45.0
+        2    232.0
+        3    323.0
+        0      NaN
+        dtype: float64
+
+        Sort values descending order
+
+        >>> s.sort_values(ascending=False)
+        3    323.0
+        2    232.0
+        7     45.0
+        6      3.0
+        5      2.0
+        4      1.0
+        1      1.0
+        0      NaN
+        dtype: float64
+
+        Sort values inplace
+
+        >>> s.sort_values(ascending=False, inplace=True)
+        >>> s
+        3    323.0
+        2    232.0
+        7     45.0
+        6      3.0
+        5      2.0
+        4      1.0
+        1      1.0
+        0      NaN
+        dtype: float64
+
+        Sort values putting NAs first
+
+        >>> s.sort_values(na_position='first')
+        0      NaN
+        4      1.0
+        1      1.0
+        5      2.0
+        6      3.0
+        7     45.0
+        2    232.0
+        3    323.0
+        dtype: float64
+        """
```

Comment:
If you find time, it would be nice to have an example that includes strings as well so that users see that sorting can apply to anything.